apps/site/assets/ts/__v3api.d.ts

```diff
@@ -63,18 +63,26 @@ export interface ParkingLotUtilization {
   typical?: number;
 }
 
-export interface PredictedOrScheduledTime {
-  delay: number;
-  scheduled_time: string[] | null;
-  prediction: Prediction | null;
-}
-
 export interface Prediction {
   time: string[];
   status: string | null;
   track: string | null;
 }
 
+export interface StopPrediction {
+  trip_id: string;
+  headsign: string;
+  route: Route;
+  prediction: PredictedOrScheduledTime;
+  train_number: string;
+}
+
+export interface PredictedOrScheduledTime {
```

Comment:
Are TS file supposed to be alphabetized?

Reply:
oops — the surrounding area in the diff suggested to me that all the interfaces/types in this specific file were already alphabetized, and so this one would break the pattern. Looking at the entire file, I can see that's not the case.

Reply:
This seems to have just been moved to a different place in the file with no changes, though — any specific reason for that, or can we leave it in its original spot for the sake of a cleaner blame?